gcp-repo/metadata.p2.inf

```diff
@@ -56,7 +56,7 @@ units.2.properties.1.value=true
 units.2.properties.2.name=org.eclipse.equinox.p2.name
 units.2.properties.2.value=YEdit
 units.2.properties.3.name=org.eclipse.equinox.p2.provider
-units.2.properties.3.value=Google Inc.
+units.2.properties.3.value=Orbit
```

Comment:
Hmm, this doesn't seem right: we're pulling this from YEdit directly.  It's not from Orbit.

Reply:
ping

Reply:
What do you think this should be?

Reply:
I think _YEdit Project_ as per the [YEdit feature](https://github.com/oyse/yedit/blob/master/org.dadacoalition.yedit.feature/feature.xml#L6).

Reply:
Done